File: String/2027.Minimum Moves to Convert String/Solution.java
class Solution {
    public int minimumMoves(String s) {
        int i = 0, ans = 0, n = s.length();
        while (i < n) {
            if (s.charAt(i) == 'X') {
                i += 3;
                ans++;
            } else {
                i++;
            }
        }
        return ans;
    }
}

File: String/2027.Minimum Moves to Convert String/Solution.py
class Solution:
    def minimumMoves(self, s: str) -> int:
        n, i, ans = len(s), 0, 0
        while i < n:
            if s[i] == 'X':
                i += 3
                ans += 1
            else:
                i += 1
        return ans